src/ImageRunModal.jsx

```diff
@@ -28,88 +28,101 @@ const units = {
     },
 };
 
-class PublishPortsForm extends React.Component {
+const PublishPort = (props) => {
```

Comment:
Style nitpick: This can be shortened to `=> (\n<React.Fragment...)`, no need for the {} and return.

Also, it's more explicit and easier to read to decompose props into the expected props (kind of like "poor (wo)man's propTypes" for functional components):

    ({ foo, bar, baz }) => { ...

Reply:
This comment is still current.